install.rst

```diff
@@ -30,17 +33,82 @@ If you use **Nix**, then you can install PostgREST from nixpkgs.
 
   nix-env -i haskellPackages.postgrest
 
-If you use Windows, you can install PostgREST using `Chocolatey <https://chocolatey.org>`_ or `Scoop <https://scoop.sh>`_.
+If you use Windows, you can install PostgREST using `Chocolatey <https://chocolatey.org/packages/postgrest>`_ or `Scoop <https://scoop.sh>`_.
 
 .. code:: bash
 
   choco install postgrest
   scoop install postgrest
 
-When a pre-built binary does not exist for your system you can :ref:`build the project from source <build_source>`.
+.. _build_source:
+
+Building from Source
```

Comment:
How about moving this to the bottom? Before the "Deploying-to-heroku" section. That way the Installation page is more straightforward.